framework/vulkan_sample.h

```diff
@@ -1337,21 +1337,6 @@ inline void VulkanSample<bindingType>::update_debug_window()
 	                                                             to_string(render_context->get_swapchain().get_format()) + " (" +
 	                                                                 to_string(vkb::common::get_bits_per_pixel(render_context->get_swapchain().get_format())) +
 	                                                                 "bpp)");
-
-	if (scene != nullptr)
-	{
-		get_debug_info().template insert<field::Static, uint32_t>("mesh_count", to_u32(scene->get_components<sg::SubMesh>().size()));
-		get_debug_info().template insert<field::Static, uint32_t>("texture_count", to_u32(scene->get_components<sg::Texture>().size()));
-
-		if (auto camera = scene->get_components<vkb::sg::Camera>()[0])
-		{
-			if (auto camera_node = camera->get_node())
-			{
-				const glm::vec3 &pos = camera_node->get_transform().get_translation();
-				get_debug_info().template insert<field::Vector, float>("camera_pos", pos.x, pos.y, pos.z);
-			}
-		}
-	}

```

Comment:
See my description of the issue as to why I removed it. tl;dr: that information was never working, in none of the samples. Adding checks  before the code would simply not run the code at all, because none of the requested components could be found.

Reply:
> because none of the requested components could be found.

And you don't expect that to ever happen?
Instead, I would prefer to have the checks extended.

Reply:
Yes, I have tested with all samples and not a single one had those working. Aside, the debug information removed with this PR is pretty useless anyway.